Description: Force groff to use tbl preprocessor
 This is Debian specific.
 No need to forward upstream.
Author: Apostolos Kefalas <sv1ljj@raag.org>
Origin: self
Forwarded: not-needed
Last-Update: 2026-05-15

--- a/doc/w_scan_cpp.1
+++ b/doc/w_scan_cpp.1
@@ -1,3 +1,4 @@
+'\" t
 .\"   IF YOU'RE  WONDERING ABOUT THIS FILE, PLS READ:
 .\"      http://www.schweikhardt.net/man_page_howto.html
 .\"
